src/components/styles/track.css

```diff
@@ -0,0 +1,3 @@
+#dashboard {
```

Comment:
I'm guessing this is copied from another file? seems like there is no component here with a 'dashboard' id. can you remove this?